Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Melee/bat_home_run.yml

```diff
@@ -0,0 +1,41 @@
+# DeltaV "Home Run Bat"
+
+- type: entity
+  name: home run bat
+  parent: BaseItem
+  id: BaseBallBatHomerun
+  description: Heavy metal bat with an extra kick.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Objects/Weapons/Melee/home_run_bat.rsi
+    state: icon
+  - type: MeleeWeapon
+    wideAnimationRotation: -135
+    damage:
+      types:
+        Blunt: 35
+        Structural: 15
+    soundHit:
+      collection: ExplosionSmall
+  - type: MeleeRequiresWield # You can't hit a home run with one hand, jimbo.
+  - type: Wieldable
+  - type: MeleeThrowOnHit
+    Speed: 30 # Triple the speed of a gorilla gauntlet, not always a good thing!
+    Lifetime: 5
+    unanchorOnHit: true
```

Comment:
will this unanchor nukes or 500 glimmer probers :warning:

Reply:
It uses the same properties as the regular gorilla glove, but lemme test

Reply:
> will this unanchor nukes or 500 glimmer probers :warning: 
> 
> 
> 
> also yaml typos:
> 
> Speed -> speed
> 
> Lifetime -> lifetime,
> 
> Enabled just remove it since its true by default

Congratulations, you just discovered a bug with the gorilla gauntlet. The gauntlet can unanchor glimmer probers at 500, which means the bat can too.

Reply:
prober should prevent unanchor via canceling an event or something on top of the shock thing when using wrench

Reply:
> prober should prevent unanchor via canceling an event or something on top of the shock thing when using wrench

I mean I can try to code it to where if glimmer is over 500 it physically cannot be unanchored

Reply:
but at the end of the day that'd also make any unanchor attempts whatsoever not do the special shock thing

Reply:
> prober should prevent unanchor via canceling an event or something on top of the shock thing when using wrench

i think i have a fix, if i add a line that checks if it's unachored somehow when glimmer is over 500, it'll explode

Reply:
gorilla gauntlet / home run bat + prober wombo combo in sec maints :trollface:

Reply:
![image](https://github.com/DeltaV-Station/Delta-v/assets/49997488/99110024-1dd7-43b9-a56b-6cecea2419b4)

I'll pr a fix for probers soon. Apparently a lot of other things could fling them too, like gravity anomalies

Reply:
For whatever reason that event isn't behaving like how I expected, so it might require a bit more work

Reply:
you need to set Cancelled = true;

Handled just means it wont use comp.Enabled

Reply:
Ah, the event is getting raised on the owner of the MeleeThrowOnHit component, not the thing being hit. Guess I'll just add a new event

Reply:
or just add another field and raise it on the thing being hit as well, less logic to maintain

Reply:
Oh yeah, guess that works too